operational/aws/lambda_functions_with_high_error_rate/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v4.0
+
+- Added logic required for "Meta Policy" use-cases
+- Updated template and readme to reflect addition of `auth_flexera`
```

Comment:
I would change this to something more human readable to a non-technical user. Something like:

- Flexera credential now required to facilitate meta policy use cases.

Reply:
Done